doc/files/package.json.md

```diff
@@ -108,8 +107,34 @@ You can check [the full list of SPDX license IDs](https://spdx.org/licenses/).
 Ideally you should pick one that is
 [OSI](http://opensource.org/licenses/alphabetical) approved.
 
-It's also a good idea to include a LICENSE file at the top level in
-your package.
+If your package is licensed under multiple common licenses, use an SPDX license
+expression syntax version 2.0 string, like this:
+
+    { "license" : "(ISC OR GPL-3.0)" }
+
+If you are using a license that hasn't been assigned an SPDX identifier, or if
+you are using an uncommon or custom license, do not include a "license" string
+in package.json. In those cases especially, but also more generally, it's a good
+idea to include a LICENSE file at the top level of the package.
```

Comment:
I am down for getting this done right the first time. `LicenseRef-LICENSE` is Magic Words, but it works.

Alas, even detecting whether a `LICENSE` file exists is messy. Sometimes `LICENSE`, other times `LICENSE.md`, `LICENSE.markdown`, `LICENSE.BSD`, &c. That's ignoring case and licenses pasted at the end of README, which has the same problem.